sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/models/PeekedMessageItem.java

```diff
@@ -41,8 +42,7 @@
     /*
      * The content of the Message.
      */
-    @JsonProperty(value = "MessageText", required = true)
-    private String messageText;
+    private BinaryData body;
```

Comment:
Is the serializer/deserialization logic going to handle this correctly?

Reply:
`PeekedMessageItemInternal` and `QueueMessageItemInteral` has been created to handle that part.
The data flow is:
receive from wire -> from XML ->PeekedMessageItemInternal -> none/base64 decoding -> PeekedMessageItem